src/prefect/cli/kubernetes.py

```diff
@@ -27,7 +28,18 @@
 
 @manifest_app.command("orion")
 def manifest_orion(
-    image_tag: str = None,
+    image_tag: str = typer.Option(
+        get_prefect_image_name(),
+        "-i",
+        "--image-tag",
+        help="The tag of a Docker image to use for the Orion.",
+    ),
+    namespace: str = typer.Option(
+        "default",
+        "-n",
+        "--namespace",
+        help="A Kubernetes namespace to create Orion in.",
+    ),
```

Comment:
@tpdorsey may be able to help us with the CLI help text here and in the new command below

Reply:
Let's address this in a follow-up